lib/sensu/server/process.rb

```diff
@@ -212,9 +216,16 @@ def aggregate_check_result(client, check)
       def store_check_result(client, check, &callback)
         @logger.debug("storing check result", :check => check)
         @redis.sadd("result:#{client[:name]}", check[:name])
+        if check[:type] == 'metric'
+          output_lines = check[:output].split("\n")
+          output = output_lines.first
+          if output_lines.size > 1 || output.length > 255
+            output = output[0..255] + "\n..."
+          end
+          check = check.merge(:output => output)
```

Comment:
We probably want to create a new object, like `check_truncated`, or we are mutating event data! 0_0